packages/create-redwood-app/template/package.json

```
@@ -15,7 +15,7 @@
     "root": true
   },
   "engines": {
-    "node": ">=14.17 <=16.x",
+    "node": ">=14.18 <=16.x",
```

My mistake; 19 is the latest minor.
```suggestion
    "node": ">=14.19 <=16.x",
```